lib/vulcan.js

```diff
@@ -177,9 +177,13 @@ Vulcan.prototype = {
   },
 
   moveToBodyMatcher: dom5.predicates.AND(
+    dom5.predicates.NOT(
+        dom5.predicates.parentMatches(
+            dom5.predicates.hasTagName('template'))),
     dom5.predicates.OR(
       dom5.predicates.hasTagName('script'),
-      dom5.predicates.hasTagName('link')
+      dom5.predicates.hasTagName('link'),
```

Comment:
Is this right? I think there might be links and scripts we don't want to move: https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types

Maybe we only move import and stylesheet?